packages/big-design/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import { ExpandLessIcon, ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { useState } from 'react';
+
+import { Flex } from '../Flex';
+
+import { StyledTitle } from './styled';
+
+export interface AccordionProps {
+  title: string;
+  onToggle?(): void;
+}
+
+export const Accordion: React.FC<AccordionProps> = ({ children, title, onToggle }) => {
+  const [isOpened, setIsOpened] = useState(false);
+
+  const toggleAccordion = () => setIsOpened(!isOpened);
+
+  const handleTitleClick = () => {
+    toggleAccordion();
+    if (onToggle) {
+      onToggle();
+    }
```

Comment:
```suggestion
    toggleAccordion();
    
    if (typeof onToggle === 'function') {
      onToggle();
    }
```

Reply:
Good catch. Checking for the type was added.